package com.api.v1.people.exceptions

class DuplicatedSsnException: RuntimeException("Given SSN is already in use.")